src/watercostdata.jl

```diff
@@ -0,0 +1,38 @@
+using DataFrames
+using RData
+
+# energy cost to lift 1000m3 by 1m
+#energycostperlift = 1.
+
+# energy cost to lift 1000m3 by 1m
+energycostperlift = 0.423133696 # in $/1000m3 /m of lift | value from CALVIN was 0.02
+
+### EXTRACTION COST
+# sw: compute relative elevation if source downhill, 0 otherwise
+# if missing information, default value is specified by naelev
+if isfile(datapath("canalextractioncost$suffix.jld"))
+    println("Loading extraction cost from saved data...")
+    canalextractioncost = deserialize(open(datapath("canalextractioncost$suffix.jld"), "r"))*energycostperlift;
+else
+    ## Optional cost for drawing down a river (environmental change)
+    # Marginal cost is $3178.73 / MG, but 92% not subject to treatment costs, so $248.53 / MG
+    canalextractioncost = 65.65 * ones(numregions)
+end
+
+if isfile(loadpath("drawdown.csv"))
```

Comment:
I've cleaned this up for datasets that don't have this file.  But I think that might be every dataset: I don't see this file at all.

Reply:
This is for colorado data only